parlai/tasks/friends/agents.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Optional
+from parlai.core.opt import Opt
+from parlai.core.teachers import DialogTeacher
+from parlai.core.params import ParlaiParser
+from .build import build
+
+import copy
+import os
+import json
+
+
+def _path(opt, *additions):
```

Comment:
Do you use the additional args? If not why keeping them?

Reply:
Ahh good catch I'll remove them